gen/aircraft.py

```diff
@@ -338,6 +343,7 @@ def __init__(
         self.joker_fuel = joker_fuel
         self.callsign = create_group_callsign_from_unit(self.units[0])
         self.custom_name = custom_name
+        self.channel = channel
```

Comment:
How does this differ from `self.intra_flight_channel`? Maybe just rename to `self.flight_channel` if you're worried about the inaccuracy of the name?